deps/spidershim/src/v8v8.cc

```diff
@@ -54,4 +54,22 @@ bool V8::IsDead() {
   return internal::gDisposed;
 }
 
+const char *V8::GetVersion() {
+  // Spidermonkey's version number does not match the format of v8's (X.X.X.X),
+  // unsure if this will be an issue yet.
+  return JS_GetImplementationVersion();
```

Comment:
This is fine!  Node [exposes](https://github.com/mozilla/spidernode/blob/312cf60c5229b50c386c56f5f993a3c6717ee18e/src/node.cc#L2934) this information as the version of the backend JS engine.